gc/base/GCExtensionsBase.hpp

```diff
@@ -706,6 +706,13 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	uintptr_t darkMatterSampleRate;/**< the weight of darkMatterSample for standard gc, default:32, if the weight = 0, disable darkMatterSampling */
 
+#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+	uintptr_t idleMinimumFree;   /** < percentage of free heap to be retained as committed, default=0 for gencon, complete tenture free memory will be decommitted */
+	uintptr_t lastGCFreeBytes;  /** < records the free memory size from last Global GC cycle */
+	uint32_t gcOnIdleRatio; /**< the percentage of allocation since the last GC allocation determines the invocation of global GC, default global GC is invoked if allocation is > 20% */
```

Comment:
It is ok for me... however keep in mind that if this variable will be involved in operations together with uintptr_t variables compiler will need to do cast 32-bit to 64-bit and back. Sure, most of the modern processors have 32-bit instructions so probably cost nothing. Also 4 bytes you are trying to save will be lost any way most likely. Also in some cases (probably not in this) it might be a reason for warning in compilation time if explicit cast is missed.